test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -389,6 +389,63 @@ public void AddPicture_sets_valid_svg_content()
         // Assert
         svgContent.Should().Contain("<svg");
     }
+    
+    [Test]
+    public void AddPicture_too_large_adds_svg_picture()
+    {
+        // See: Issue #683 Large-dimension SVG files lead to error opening in PowerPoint
+
+        // Arrange
+        var pres = new Presentation();
+        var shapes = pres.Slides[0].Shapes;
+        var image = TestHelper.GetStream("test-vector-image-large.svg");
+        image.Position = 0;
+
+        // Act
+        shapes.AddPicture(image);
+
+        // Assert
+        shapes.Should().HaveCount(1);
+        var picture = (IPicture)shapes.Last();
+        picture.ShapeType.Should().Be(ShapeType.Picture);
+
+        // These values are the intrinsic size of the test image
+        picture.Height.Should().BeLessThan(2400);
+        picture.Width.Should().BeLessThan(2400);
+        pres.Validate();
+    }
+
+    [Test]
+    public void AddPicture_too_large_adds_picture()
+    {
+        // See: Issue #683 Large-dimension PNG files lead to error opening in PowerPoint
```

Comment:
Please remove this comment.

Although such comments in tests might be useful, e.g., for traceability by providing context and answering why this test was created, I believe the _**unit**_-test should be as self-sufficient as possible. Things like the _**name**_ of the test, keeping it _**small**_, and having _**fewer assertions**_ should ensure user with full information about what is being test without external links.

Reply:
Ok, is done